lib/appraisal/dependency_list.rb

```diff
@@ -1,4 +1,5 @@
 require 'appraisal/dependency'
+require "set"
```

Comment:
Probably should remain consistent with single vs. double quotes

Reply:
I tried that at first, but the CI linting bot yield at me for that, so I changed it to double quotes.

Looking at recent changes in the repo, double quotes seems to be the new pattern.